packages/gateway/src/features/registration/root-resolvers.ts

```diff
@@ -637,7 +596,7 @@ export const resolvers: GQLResolver = {
       return taskEntry.resource.id
     },
     async confirmRegistration(_, { id }, { headers: authHeader }) {
-      if (!inScope(authHeader, ['record.confirm-registration'])) {
+      if (!inScope(authHeader, ['record.confirm-registration'] as unknown as Scope[])) {
```

Comment:
@noah-brunate https://github.com/opencrvs/opencrvs-core/blob/a7fd62a51a98e39361ac86cd9b0c2e0811554d2a/packages/commons/src/authentication.ts#L29 The scope should actually be there so no casting `as unknown as Scope[]` should be needed